src/video.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import {useDisableBorderRadius} from './image';
+import {createUseStyles} from './jss';
+import {combineRefs} from './utils/common';
+
+export type AspectRatio = '1:1' | '16:9' | '12:5';
+
+export const RATIO = {
+    '1:1': 1,
+    '16:9': 16 / 9,
+    '12:5': 12 / 5,
+};
```

Comment:
I considered, instead of a fixed set of ratios, to accept a number. Anyway, extending this is easy